build_tools/github_actions/runner/gcp/remove_idle_runners.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# Due to limitations in MIGs, we quite frequently want to manually bring down
+# instances that aren't currently running a job. This script automates that.
+# Note that there is totally the potential for a race here. Sorry.
```

Comment:
What sort of race? A runner picks up a job after being identified as idle? (or the other way around - a runner goes idle after being identified as busy)

Reply:
The former. Expanded the comment